contracts/dynamic-caller-contract/src/contract.rs

```diff
@@ -24,6 +24,8 @@ struct CalleeContract {
     address: Addr,
 }
 
+// When `user_defined_mock = true`, user can and must write mock implement for test
+// of the trait for specified struct with `#[cfg(not(target_arch = "wasm32"))]`.
```

Comment:
which part "with" modifies is ambiguous. rewrite to 

... mock implement with `#[cfg(not(target_arch = "wasm32"))]` of the trait for ...

Reply:
Thanks!! That is certainly true. Fixed.